extensions/socketio/package.json

```diff
@@ -21,6 +21,7 @@
   "license": "MIT",
   "dependencies": {
     "@loopback/boot": "^3.1.0",
+    "@loopback/context": "^3.13.0",
     "@loopback/core": "^2.12.0",
```

Comment:
The convention is to use `@loopback/core`, which re-exports artifacts from `@loopback/context`. We also should move `@loopback/core` to `peerDependencies` and `devDependencies`.

Reply:
See an example at https://github.com/strongloop/loopback-next/blob/master/extensions/graphql/package.json#L33

Reply:
sure @raymondfeng , i'll make the changes

Reply:
Please also update package.json so that `@loopback/core` is moved to `peerDependencies` and `devDependencies` following what's there in https://github.com/strongloop/loopback-next/blob/master/extensions/graphql/package.json#L33.

Reply:
@raymondfeng made the changes